commerce-account-admin-web/src/main/java/com/liferay/commerce/account/admin/web/internal/display/context/CommerceAccountAdminDisplayContext.java

```diff
@@ -40,16 +42,32 @@ public CommerceAccountAdminDisplayContext(
 		ModelResourcePermission<CommerceAccount>
 			commerceAccountModelResourcePermission,
 		CommerceAccountService commerceAccountService,
+		CommerceAddressService commerceAddressService,
 		RenderRequest renderRequest,
 		UserFileUploadsConfiguration userFileUploadsConfiguration) {
 
 		super(
 			commerceAccountModelResourcePermission, commerceAccountService,
 			renderRequest);
 
+		_commerceAddressService = commerceAddressService;
 		_userFileUploadsConfiguration = userFileUploadsConfiguration;
 	}
 
+	public List<CommerceAddress> getBillingCommerceAddresses()
+		throws PortalException {
+
+		CommerceAccount commerceAccount = getCommerceAccount();
+
+		if (commerceAccount == null) {
+			return null;
```

Comment:
So.... we're looping on this value in the JSP. We should return `Collections.emptyList` or it's gonna throw an NPE.

Reply:
I think the loop should avoid the NPE. But I do like returning an empty list better.